src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -1352,6 +1353,9 @@ public void checkout(Run<?, ?> build, Launcher launcher, FilePath workspace, Tas
 
         // Needs to be after the checkout so that revToBuild is in the workspace
         try {
+            String shortMessage = getCommitMessage(listener, git, revToBuild);
+            listener.getLogger().println("Commit message: \"" + shortMessage + "\"");
+            environment.put(GIT_COMMIT_TITLE, shortMessage);
```

Comment:
@MarkEWaite In this(https://github.com/jenkinsci/git-plugin/pull/1049/files#r583842074) pull request you mentioned with this enviroment variable is set only for the process.
I want to understand what is the scope of adding a variable to EnvVars is? Since it is being added in git plugin would it be available to just the checkout step or it will be available in subsequent steps like build etc. ? From the bug description I believe we want to use this env variable to check if we want to run the build step based on the commit message. So would this env variable be available in the pipeline script in the build step with this ?